assets/js/hocs/test/with-attributes.js

```diff
@@ -88,20 +88,16 @@ describe( 'withAttributes Component', () => {
 			expect( props.expandedAttribute ).toBe( 1 );
 		} );
 
-		it( 'getTerms is called on mount if there was an attribute selected', ( done ) => {
-			const renderer = TestRenderer.create(
+		it( 'getTerms is called on mount if there was an attribute selected', async () => {
+			const renderer = await TestRenderer.create(
```

Comment:
`TestRenderer` doesn't return a promise so I don't think this is the way to go here. You _should_ be able to just add the `async` here and leave the existing `getAttributesPromise.then`  logic in place.

Reply:
fixed in d325e3e